srcpkgs/swaync/template

```diff
@@ -0,0 +1,16 @@
+# Template file for 'swaync'
+pkgname=swaync
+version=0.7.2
+revision=1
+wrksrc=SwayNotificationCenter-${version}
+build_style=meson
+hostmakedepends="vala glib-devel pkg-config"
+makedepends="gobject-introspection gtk+3-devel gtk-layer-shell-devel
+json-glib-devel libhandy1-devel scdoc"
+depends="dbus"
+short_desc="Simple notification daemon with a GTK gui"
+maintainer="Akhil Jalagam <akhil.j@gmail.com>"
+license="GPL-3.0-or-later"
```

Comment:
Reasonably sure this should be "GPL-3.0-only"; I searched through the source tree for "or later" and didn't find any matches.

Reply:
what about "or any later"? that's the usual text suggested by the licence

Reply:
Only in the LICENSE file when it's explaining what "or any later version" means.

Reply:
![image](https://user-images.githubusercontent.com/21128035/197375288-9ee23cc6-940d-47a1-a5b1-50fb1c1c5a79.png)
I will use GPL-3.0+

Reply:
using 'GPL-3.0-only' now.